### ISSUE TITLE
Build FE template for Content Type: Checklist 

### ISSUE THREAD

**Issue by brianalloyd** [department-of-veterans-affairs/va.gov-team#13497]
### **Issue Description**
Given CMS templates have been built for the "Multiple Q/A" component
I want to begin building the FE template so that this component is successfully used on VA.gov
Separating these into three separate ticket based on content type component. i.e. checklist, video, images. 

### The CMS templates for this ticket:

Images (Related CMS ticket #2453 )

### **Tasks**

- [ ] What work is necessary for this story to be completed?

### Acceptance Criteria
- [ ] This layer of the FE template (meat layer) is complete for this content type
- [ ] Need to groom, review and estimate   

### Design

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/wip-files/VA-LCT-Checklist.png

**Comment by kelsonic** [comment]
We should have all the data we need for this from the CMS:

![image](https://user-images.githubusercontent.com/12773166/93497676-edcf6e80-f8cd-11ea-94c0-b8980e8cfb14.png)

**Comment by ncksllvn** [comment]
This checklist seems like it's just a different style of bulleted list that might be friendly for printing. I don't see it on the design system or forms system. It's even different from our typical checkbox input, https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/errorablecheckbox/.

I don't want us to get blocked on a design system convo. But @kelsonic, what do you think of building this as a list with a custom bullet? It's almost [list-style-type: circle](https://www.w3schools.com/cssref/pr_list-style-type.asp). We could possibly use [list-style-image](https://www.w3schools.com/cssref/pr_list-style-image.asp).

**Comment by kelsonic** [comment]
Eesh, I feel like that's risky to invest time in making it not a checkbox, because I know the conversation will come up "why are we showing a checkbox when a user can't interact with it?" The probability for design to change that spec seems very high, may be best to just implement an <li> tag or something for now until we get more certainty CC @jenniferlee-dsva

**Comment by ncksllvn** [comment]
Makes sense to me - sounds very MVP! But it might be a core part of this page-type, so I could see it either way.

@jenniferlee-dsva this is boiling down to - should we proceed with building the empty boxes to use as the bullets in the list, or should we proceed with just a typical bulleted list?

**Comment by jenniferlee-dsva** [comment]
Hello - Yes, this template was  intended to allow users to check the box -  in the browser and by printing it out as a checklist. 

1/ The checkboxes should be checkable on the web page
2/ When the user prints, the checkboxes should render as checkboxes. including if the user checked it, should render as a checked box.

@rtwell to chime in. I believe that this is a new design, and is leading the design council stuff for new designs.
 
**Just a reminder - MVP 1.0 list of article pages are :**
1. Question and answer
2. Multiple FAQs
3. Step-by-step
4. About

The Checklist will have to be completed for  1.1 December release. So if we need to work on this minimally now and troubleshoot/refine later, that's acceptable. Please just make sure to capture any additional refinement tasks in a ticket, so we're not scrambling last minute.

**Comment by ncksllvn** [comment]
@MickinSahni @brianalloyd per Jen's comment, was this added into iteration 1 by mistake? Also, I'm wondering if we have an issue already created for the About template, and if we need to update the Iteration 1 Epic to include the About template, https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/12365

**Comment by ncksllvn** [comment]
>  So if we need to work on this minimally now and troubleshoot/refine later, that's acceptable. Please just make sure to capture any additional refinement tasks in a ticket, so we're not scrambling last minute.

I went ahead and moved the barebones template forward to code review. We'll definitely need to revisit/sync on this to identify refinements.

**Comment by kelsonic** [comment]
PR validated on http://preview-staging.vfs.va.gov/preview?nodeId=6924

Closing as AC were completed! 🎉

**Comment by ncksllvn** [comment]
@jenniferlee-dsva Is this now included in the first MVP? I see it included in the [product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/product-outline.md#high-level-requirements-for-mvp) but didn't think it was

**Comment by jenniferlee-dsva** [comment]
Hi Nick - it is part of mvp 1.1 - so mvp in general. (That's confusing! Sorry!)

Please use this for the phases of mvp's: 
https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/content/tier-2-content-IA-and-design/learning-center-mvp/mvp-phases-to-launch
